### PR TITLE
Fix schema.json

### PR DESCRIPTION
**Pull request by cPu1**
### Description

https://github.com/weaveworks/eksctl/pull/4912 fixed the doc link in godoc comments, but the schema wasn't generated from it. This changelist regenerates the schema.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [x] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: .github/workflows/exclude-file.txt [hunk 1]
```diff
@@ -11,4 +11,4 @@ config.site_url
 eks.amazonaws.com
 raw.githubusercontent.com
 creating-and-managing-clusters
-schema.js
\ No newline at end of file
+schema.js
```
Comment (Himangini) [inline .github/workflows/exclude-file.txt:14]: when did this get removed? 🤔

Comment (cPu1) [reply .github/workflows/exclude-file.txt:14]: It isn't removed, this changelist adds a newline after it. The diff is a little confusing.

File: pkg/apis/eksctl.io/v1alpha5/assets/schema.json [hunk 2]
```diff
@@ -300,7 +300,7 @@
           },
           "type": "array",
           "description": "service accounts to create in the cluster. See [IAM Service Accounts](/usage/iamserviceaccounts/#usage-with-config-files)",
-          "x-intellij-html-description": "service accounts to create in the cluster. See <a href=\"/iamserviceaccounts/#usage-with-config-files\">IAM Service Accounts</a>"
+          "x-intellij-html-description": "service accounts to create in the cluster. See <a href=\"/usage/iamserviceaccounts/#usage-with-config-files\">IAM Service Accounts</a>"
```
[the remark this answers is not in the document]
Comment (cPu1) [reply pkg/apis/eksctl.io/v1alpha5/assets/schema.json:303]: The link is incorrect there in the `x-intellij-html-description` field.

[the remark this answers is not in the document]
Comment (Himangini) [reply pkg/apis/eksctl.io/v1alpha5/assets/schema.json:303]: ah ok nice spot completely missed the `x-intellij-html-description` field 😓

File: .github/workflows/exclude-file.txt (hunk 1 above)
Comment (Himangini) [reply .github/workflows/exclude-file.txt:14]: thanks makes sense 👍🏻

**Review by Himangini — APPROVED** [review]
👍🏻

**Review by aclevername — APPROVED** [review]

**Pull request merged**